### ISSUE TITLE
Add Implementation section

### ISSUE THREAD

**Issue by alagoutte** [pcapng/pcapng#29]
May be add a Implementation section  ( https://tools.ietf.org/html/rfc6982 ) ?

for start can be add a link to https://github.com/pcapng/pcapng/wiki/Implementations (or copy the table)

**Comment by hadrielk** [comment]
Yes I think just add a URL to the wiki - there are more implmentations we can add, and it would be a pain to add them as edits to the draft via pull requests.
